src/Common/src/CoreLib/System/Text/ValueStringBuilder.cs

```diff
@@ -20,6 +21,12 @@ public ValueStringBuilder(Span<char> initialBuffer)
             _chars = initialBuffer;
             _pos = 0;
         }
+        public ValueStringBuilder(int initialCapacity)
```

Comment:
nit: add new line between the ctors

Reply:
[Done](https://github.com/dotnet/corefx/pull/27250/commits/b1b17d2a0e58e863e86440651b188ac3aab86c77#diff-1a641bcf15d0f95ae2b8a25f157ecd51R24), thanks!